test/testlib.sh

```diff
@@ -431,3 +431,25 @@ verify_all_restored_data() {
   # verify common data
   verify_common_data
 }
+
+setup_moreutils_parallel() {
+  # CI servers may have moreutils parallel and GNU parallel installed. We need moreutils parallel.
+  for x in \
+      /usr/bin/parallel.moreutils \
+      /usr/bin/parallel_moreutils \
+      /usr/bin/moreutils.parallel \
+      /usr/bin/moreutils_parallel \
+      ; do
+        if [ -x "${x}" ]; then
+            ln -sf "${x}" "$ROOTDIR/test/bin/parallel"
+            break
+        fi
+  done
+  unset x
```

Comment:
`x` could be a local variable and then you wouldn't need to `unset` here.

Reply:
Fixed